src/dist/replication/meta_server/meta_server_failure_detector.cpp

```diff
@@ -83,6 +85,23 @@ void meta_server_failure_detector::on_worker_connected(rpc_address node)
 
 bool meta_server_failure_detector::get_leader(rpc_address *leader)
 {
+    FAIL_POINT_INJECT_F("meta_server_failure_detector_get_leader", [leader](dsn::string_view str) {
+        auto pos = str.find("#");
```

Comment:
Could you please add comments to explain `leader` format? because leader format for unit test `meta_service_test.cpp` is different from structure `rpc_address`, I just can't understand it before I read unit tests.